INSTALL.md

```diff
@@ -0,0 +1,138 @@
+
+# Building and Installation
+
+## Dependencies
+
+Ensure the development version of the dependencies are installed. These are as follows:
+
+* [libhdbpp](https://github.com/tango-controls-hdbpp/libhdbpp) - (libhdb++6). Build and install according its instructions.
+* [Datastax CPP Driver](https://github.com/datastax/cpp-driver), version 2.2.1.
+* Tango Controls 9.2.5a.
+* omniORB release 4 - libomniorb4 and libomnithread
+* libzmq - libzmq3-dev or libzmq5-dev
+
+If they have not been installed in a standard location, then use standard CMake flags below to inform the build where to search for them.
+
+The library can be built with Tango Controls install via debian package or source distribution. Supported Tango Controls release is currently 9.2.5a. Ensure build flags are used if tango is installed to a custom location.
+
+Toolchain dependencies:
+
+* CMake 3.0.0 or greater is required to perform the build.
+
+### Datastax CPP Driver
+
+Download the [DataStax Cassandra C++ Driver](https://github.com/datastax/cpp-driver) from github and install. This is dependent on having [Libuv](https://github.com/libuv/libuv_) built and installed on the system. Datastax provide detailed [instructions](http://datastax.github.io/cpp-driver/topics/building/) to build the C++ Driver.
```

Comment:
Datastax provide => provides

Reply:
Libuv link is broken (The extra '_' should be removed).

Reply:
Fixed